connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/avro/AvroRecordReader.java

```diff
@@ -68,7 +68,7 @@ private Object serializeRow(GenericRecord record) {
     List<Schema.Field> fields = actualSchema.getFields();
     Object[] row = new Object[columnNames.size()];
     for (Schema.Field field : fields) {
-      int colIndex = columnNames.indexOf(field.name());
+      int colIndex = columnNames.indexOf(field.name().toLowerCase());
```

Comment:
can we add test case for external table bq column name (not just pesudo column) have upper case?

Reply:
Good point. I made this small change to address that: https://github.com/GoogleCloudDataproc/hive-bigquery-connector/pull/98/commits/2b0e33d76ec5913c2b28ded199ef5d662f844152